spring/rocketcorner/src/main/java/com/example/rocketcorner/controller/UserRoutes.java

```diff
@@ -75,21 +76,43 @@
 
     @PostMapping("/login")
```

Comment:
we should also check if they're admin here and maybe return a user id of 1 or something. maybe make Admin.getID and set it to 1 and return that